ui/src/config/section/image.js

```diff
@@ -268,7 +268,7 @@ export default {
           dataView: true,
           show: (record, store) => {
             return (['Admin'].includes(store.userInfo.roletype) || // If admin or owner or belongs to current project
-              (record.domainid === store.userInfo.domainid && record.account === store.userInfo.account) ||
+              (record.domainid === store.userInfo.domainid && record.account === store.userInfo.account && record.isextractable) ||
               (record.domainid === store.userInfo.domainid && record.projectid && store.project && store.project.id && record.projectid === store.project.id)) &&
               !(record.account === 'system' && record.domainid === 1) &&
               record.isready
```

Comment:
@Spaceman1984 shouldn't `isextractable` condition be added with `isready` irrespective of same account or not?

Reply:
No - admins can download all ISOs